.env

```diff
@@ -0,0 +1,12 @@
+SECRET_KEY=&>\tLe8T,Jj?PU4R)a@!<)I\n+GfhtiKTok8S8&jnAzj{&%FBGW-.^b(xr]6yr
+
+DEBUG=True
+
+DEVELOPMENT_MODE = 'local'
```

Comment:
Is there a reason for keeping the quote? if not it I hope it would be edited to: "DEVELOPMENT_MODE=local"